packages/apps-config/src/api/spec/clover.ts

```diff
@@ -0,0 +1,65 @@
+// Copyright 2017-2021 @polkadot/apps-config authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+/**
```

Comment:
```suggestion

// structs need to be in order
/* eslint-disable sort-keys */

/**
```

The linting is great elsewhere, but you don't want it to start complaining about structs (or worse) "fix" them since they need to be in the defined order.

Reply:
Cool, let me fix it.